### ISSUE TITLE
Fix `User`/`Project` models, due to unsupported UUID4 column in sqlite

### ISSUE THREAD

**Issue by jluethi** [fractal-analytics-platform/fractal-server#560]
After one project is deleted, trying to create a new project leads to an error for that user:
```
(sqlite3.IntegrityError) UNIQUE constraint failed: linkuserproject.project_id, linkuserproject.user_id\n[SQL: INSERT INTO linkuserproject (project_id, user_id) VALUES (?, ?)]\n[parameters: (6, '9cf73131431041a998ef494fdc99c230')]\n(Background on this error at: https://sqlalche.me/e/14/gkpj)
```

It still works for other users. But it appears deleting projects puts the user on that server into a compromised state.
Tested both on the UZH deployment and on a local server running on my Mac. But were running 1.1.0a1 version of the server.

If there is a good fix for it, let's fix this soon. Otherwise, let's deactivate the project delete functionality for the time being.

cc @adrtsc I could also reproduce your error here. We'll track progress on it in this issue. Thanks for finding it!

**Comment by jluethi** [comment]
Other operations still work for the user though, e.g. creating new workflows

**Comment by jluethi** [comment]
Hmm, but I can't always reproduce this error. If I delete the first project, it all seems ok. But if I delete e.g. the third project, then I get this error after trying to create new ones. I have a minimal database that reproduces this now, I'll share it via slack :)

**Comment by tcompa** [comment]
This is likely due to sqlite (https://github.com/fractal-analytics-platform/fractal-server/pull/567#issuecomment-1471495093).

**Comment by tcompa** [comment]
Great, thanks @mfranzon!

I'll collect all user-related issues in a summary one.

**Comment by tcompa** [comment]
Closed with #660
